src/vm/frames.h

```diff
@@ -2464,20 +2459,19 @@ class ComPrestubMethodFrame : public ComMethodFrame
 
 #endif // FEATURE_COMINTEROP
 
-
 //------------------------------------------------------------------------
 // This frame protects object references for the EE's convenience.
 // This frame type actually is created from C++.
+// There is a chain of GCFrames on a Thread, separate from the
```

Comment:
It would be nice to move all GCFrame/GCPROTECT related stuff to a separate file (it can be a follow up change).

Reply:
Makes sense, I was also considering doing that, but I didn't want to clutter the review.